content/guides/user-guides/delfin/driver-matrix-support/dell-emc-vmax/_index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Dell EMC VMAX
+menuTitle: Dell EMC VMAX 
+description: "Dell EMC VMAX Support Matrix"
+weight: 20
+disableToc: false
+tags: ["user guide", "delfin", "drivers"]
+---
+
+### Model and Versions Supported
+
+|Models   | Firmware Versions   |
```

Comment:
Please populate the content for the Dell EMC Vmax driver support.

Reply:
NA, it is just a template

Reply:
This is not the template. this document will hold the values for Dell EMC VMAX driver.

Reply:
As per our discussion #144 has created to fill all the data